pkg/migrations/sql/versions/dev/0.7.2-dev/4-data_retention.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO SCHEMA_CATALOG.default(key, value)
+VALUES ('span_retention_period', (30 * INTERVAL '1 days')::text);
```

Comment:
This is effectively a breaking change, as it will result in data deletion for setups running longer than 30 days. Interestingly this is one of those cases where we would prefer the migration script to do something other than the preinstall.

Reply:
Agreed. Yeah, since this is beta, I'm thinking it's okay.

Reply:
Yes, we should mention in the release notes and indicate to users how they can extend their retention.

Reply:
Do I mention this [here](https://github.com/timescale/promscale/releases/tag/untagged-a5f5b8393fcc39093e88) for now? When we have the CHANGELOG stuff added, I would just put it in the changelog, correct?

Reply:
you can put it in https://github.com/timescale/promscale/releases/tag/untagged-f8c325bead62cae08400